conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosIndexer.scala

```diff
@@ -352,176 +352,86 @@ class TezosIndexer(
       val (blocks, accountUpdates) =
         results.map {
           case (block, accountIds) =>
-            block -> accountIds.taggedWithBlock(
-                  block.data.hash,
-                  block.data.header.level,
-                  Some(block.data.header.timestamp.toInstant),
-                  TezosOptics.Blocks.extractCycle(block),
-                  TezosOptics.Blocks.extractPeriod(block.data.metadata)
-                )
+            block -> accountIds.taggedWithBlockData(block.data)
         }.unzip
 
       for {
         _ <- db.run(TezosDb.writeBlocksAndCheckpointAccounts(blocks, accountUpdates)) andThen logBlockOutcome
         _ <- tnsOperations.processNamesRegistrations(blocks).flatMap(db.run)
-        votingData <- processVotesForBlocks(results.map { case (block, _) => block })
-        rollsData = votingData.map {
-          case (block, rolls) => block.data.hash -> rolls
-        }
-        delegateCheckpoints <- processAccountsForBlocks(accountUpdates, rollsData) // should this fail, we still recover data from the checkpoint
-        _ <- processBakersForBlocks(delegateCheckpoints) // same as above
-        _ <- processBlocksForGovernance(blocks, votingData)
-        _ <- updateBakers(blocks)
+        bakersData <- nodeOperator.getBakerRollsForBlocks(blocks)
```

Comment:
Perhaps this should be `bakerRollsData`?

Reply:
Is `rollsData` enough? it's a local name, after all